library/src/main/java/com/chuckerteam/chucker/internal/ui/MainViewModel.kt

```diff
@@ -9,10 +9,16 @@ import com.chuckerteam.chucker.internal.data.entity.HttpTransactionTuple
 import com.chuckerteam.chucker.internal.data.entity.RecordedThrowableTuple
 import com.chuckerteam.chucker.internal.data.repository.RepositoryProvider
 import com.chuckerteam.chucker.internal.support.NotificationHelper
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.plus
 
 internal class MainViewModel : ViewModel() {
 
     private val currentFilter = MutableLiveData<String>("")
+    private val backgroundScope = CoroutineScope(Dispatchers.IO) + Job()
```

Comment:
You could use `viewModelScope`. I don't remember if some additional extension library is needed for it.

Reply:
right. agree. why couldn't i remember that !